lib/crypto/src/crypto.erl

```diff
@@ -47,533 +56,572 @@
+]).
+
 -export([cmac/3, cmac/4]).
 
 -deprecated([{poly1305, 2, "use crypto:mac/3 instead"}]).
+
 -export([poly1305/2]).
 
--deprecated([{stream_init, '_',
-              "use crypto:crypto_init/3 + crypto:crypto_update/2 + "
-              "crypto:crypto_final/1 or crypto:crypto_one_time/4 instead"},
-             {stream_encrypt, 2, "use crypto:crypto_update/2 instead"},
-             {stream_decrypt, 2, "use crypto:crypto_update/2 instead"},
-             {block_encrypt, 3,
-              "use crypto:crypto_one_time/4 or crypto:crypto_init/3 + "
-              "crypto:crypto_update/2 + crypto:crypto_final/1 instead"},
-             {block_encrypt, 4, 
-              "use crypto:crypto_one_time/5, crypto:crypto_one_time_aead/6,7 "
-              "or crypto:crypto_(dyn_iv)?_init + "
-              "crypto:crypto_(dyn_iv)?_update + crypto:crypto_final instead"},
-             {block_decrypt, 3,
-              "use crypto:crypto_one_time/4 or crypto:crypto_init/3 + "
-              "crypto:crypto_update/2 + crypto:crypto_final/1 instead"},
-             {block_decrypt, 4,
-              "use crypto:crypto_one_time/5, crypto:crypto_one_time_aead/6,7 "
-              "or crypto:crypto_(dyn_iv)?_init + "
-              "crypto:crypto_(dyn_iv)?_update + crypto:crypto_final instead"}
-            ]).
--export([stream_init/2, stream_init/3,
-         stream_encrypt/2,
-         stream_decrypt/2,
-         block_encrypt/3, block_encrypt/4,
-         block_decrypt/3, block_decrypt/4
-        ]).
--deprecated_type([{retired_cbc_cipher_aliases, 0, "Use aes_*_cbc or des_ede3_cbc"},
-                  {retired_cfb_cipher_aliases, 0, "Use aes_*_cfb8, aes_*_cfb128 or des_ede3_cfb"},
-                  {retired_ctr_cipher_aliases, 0, "Use aes_*_ctr"},
-                  {retired_ecb_cipher_aliases, 0, "Use aes_*_ecb"}
-                 ]).
+-deprecated([
+    {stream_init, '_',
+        "use crypto:crypto_init/3 + crypto:crypto_update/2 + "
+        "crypto:crypto_final/1 or crypto:crypto_one_time/4 instead"},
+    {stream_encrypt, 2, "use crypto:crypto_update/2 instead"},
+    {stream_decrypt, 2, "use crypto:crypto_update/2 instead"},
+    {block_encrypt, 3,
+        "use crypto:crypto_one_time/4 or crypto:crypto_init/3 + "
+        "crypto:crypto_update/2 + crypto:crypto_final/1 instead"},
+    {block_encrypt, 4,
+        "use crypto:crypto_one_time/5, crypto:crypto_one_time_aead/6,7 "
+        "or crypto:crypto_(dyn_iv)?_init + "
+        "crypto:crypto_(dyn_iv)?_update + crypto:crypto_final instead"},
+    {block_decrypt, 3,
+        "use crypto:crypto_one_time/4 or crypto:crypto_init/3 + "
+        "crypto:crypto_update/2 + crypto:crypto_final/1 instead"},
+    {block_decrypt, 4,
+        "use crypto:crypto_one_time/5, crypto:crypto_one_time_aead/6,7 "
+        "or crypto:crypto_(dyn_iv)?_init + "
+        "crypto:crypto_(dyn_iv)?_update + crypto:crypto_final instead"}
+]).
+
+-export([
+    stream_init/2, stream_init/3,
```

Comment:
Why are some kept in the same line? Is it possible to use an option to further "expand" them?

Reply:
> functions that have the same name, but different arity
Great. I thus understand the rationale. I like it. Surely grouping per name (e.g. sorted, as you say) would probably benefit other use cases, but that might be a future option too.

Reply:
Great to hear you like it :)

Yes we have decided to leave sorting exports to another tool that can be used in conjunction with `erlfmt`.